### PR TITLE
Add note about which version embedded tool is implemented

### PR DESCRIPTION
**Pull request by techknowlogick**
as title

### CODE REVIEW

File: docs/content/doc/advanced/cmd-embedded.en-us.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ inside the `custom` directory (see [Customizing Gitea]({{< relref "doc/advanced/
 To obtain a copy of the embedded resources ready for editing, the `embedded` command from the CLI
 can be used from the OS shell interface.
 
+The embedded data extraction tool is included in Gitea versions 1.12 and above.
```
Comment (jolheiser) [inline docs/content/doc/advanced/cmd-embedded.en-us.md]: Maybe something to grab attention more?
```suggestion
**NOTE:** The embedded data extraction tool is included in Gitea versions 1.12 and above.
```

**Review by jolheiser — APPROVED** [review]
Small nit, otherwise LGTM

**Review by guillep2k — APPROVED** [review]
It's a difficult choice since "is included in versions 1.12 and above" implies that those versions are already released (our downloads page doesn't have an `1.12` folder). I thought about this when I wrote the docs for that and for other features.

I understand that we can't start adding "when released" everywhere, because that becomes obsolete easily. One possible solution for this is having different repositories for code and docs, so docs can follow its own release/backport pattern.

As for this case in particular, this message is good enough.

**Comment by techknowlogick** [comment]
@guillep2k sadly we've tried separate repo for docs before, and it caused no docs to be added at all.

**Comment by guillep2k** [comment]
Other possible ways out of this conundrum:

- Have some kind of dynamic setting in Hugo to `{{if}}` out comments that are not applicable.
- Don't use `master` as a directory in the downloads page, but `1.12-development` instead.

**Pull request merged**